### DISCUSSION TITLE
Installation fails on Win 10 x64

### DISCUSSION THREAD

**Discussion by w9ran** [AlexandreRouma/SDRPlusPlus#581] [Q&A]
Installation fails due to VCRUNTIME140_1.dll is missing.  I found and tried to install it but it won't because a newer version is already installed on my PC.  I hesitate to remove it to roll back to the required version for fear other things will break.   Is there a way around this?
Thanks, Bob

**Comment by AlexandreRouma** [comment]
You can, and usually have to, install both versions side by side.
Make sure to download the official installer from microsoft and not other websites that offer the DLL by the way

**Reply by w9ran** [reply]
I'm still unable to get SDR++ to install.  I started by trying to install what seems to be the proper redistributable but Windows refused unless I uninstalled an older version first.  I found about a dozen versions back to 2008, but the package I found at the Microsoft site was supposed to replace everything from 2015 on, so I uninstalled the 2017 and 2015 versions.  Yet the same error occurs.   Here is the Microsoft recommended package:

https://docs.microsoft.com/en-US/cpp/windows/latest-supported-vc-redist?view=msvc-170

I don't know how to convince Windows to allow two versions to be installed at the same time as their installer won't proceed as long as the older version is present.   Do I need to uninstall ALL older versions?   If so, what else will break?

At this time the older versions are gone and the above which is supposed to replace them won't install.  I'll attach the error log, which isn't very helpful to me at least.
[dd_vcredist_amd64_20220206182156.log](https://github.com/AlexandreRouma/SDRPlusPlus/files/8011530/dd_vcredist_amd64_20220206182156.log)


Don't know why I'm having such trouble with this as it's a standard Windows 10 x64 installation, updated.

Thanks for any suggestions,
Bob

**Reply by w9ran** [reply]
Well I guess it's beyond my ability to make it work.  Intel says "Resolution Intel® HD Graphics 2000 supports up to Windows 8.1.  There are no drivers for Windows 10."    I've installed Win64_152820.exe which is the Intel HD2000 driver package and it installs OK but there is no GL support beyond 1.1 afterward.     I've given up on tat PC and have installed SDR++ on another PC which provides OpenGL support and it loads and runs but with problems using Airspy R2.   I would appreciate your help with the following:   

1.  Speaker icon is "off" and I cannot get it to turn on or to increase the volume slider.    (Audio sink is correctly configured to Audio, internal speaker/headphones, 48K, and the volume slider in the side panel is halfway.  No sound at all even though signals are visible on the spectrum display.

2.  Mode is stuck on WFM and cannot be changed.  I can open the Radio menu but that's all - there is no response to any of the other buttons or controls in "Radio"  

3.  SDR# will play normally using the same radio source (Airspy R2) and audio selections.

Thanks,
Bob



Bob

**Comment by w9ran** [comment]
Actually I fixed it.  Don't know why but I just blew the installation away and re-installed from the zip file and now everything works.  Sorry for the SNR.
